resources/airflow.rb

```diff
@@ -1722,6 +1667,60 @@ def self.create_infil_mech_vent_objects(model, runner, building, infil, mech_ven
       infil_program.addLine("Set Qn = #{building.living.ACH * UnitConversions.convert(building.infilvolume, "ft^3", "m^3") / UnitConversions.convert(1.0, "hr", "s")}")
     end
 
+    if mech_vent.type == Constants.VentTypeBalanced and mech_vent.sensible_effectiveness > 0 and mech_vent.whole_house_cfm > 0
+      # ERV/HRV; balanced systems without energy recovery are modeled via infiltration actuated EMS
```

Comment:
This comment needs to be updated since *all* mech vent systems are now modeled via EMS. It seems we could refer to "EMS airflow model" and "EMS load model"?

Reply:
Also, I suggest adding a comment describing why we are not using the E+ ERV object.